### DISCUSSION TITLE
Need someone or a few people to take over this repo

### DISCUSSION THREAD

**Discussion by iklein99** [iklein99/homebridge-smartthings#270] [Announcements]
Hi all.  Unfortunately, I no longer have the time to work on this repo - at least not at the rate I would like to.  I'm looking for a volunteer or 2 to participate and eventually take over maintaining the repo.  Let me know if you are interested.

**Comment by kshartman** [comment]
Hi, I have used this for a long time.  I pay you with paypal for the webhook service you run (for each home).  Will that still be running and maintained?  I don't think the server code is in this repo?  This plugin is the only thing that kept me with smartthings instead of moving entirely to hubitat or hab.  Thanks!

**Comment by iklein99** [comment]
I plan to maintain the server.  It is used to handle events from SmartThings. You only need it if you use things like motion sensors and buttons.  Otherwise you can cancel that and remove the webhook token from your config and the plugin will pill for updates which is slower and generates more traffic.  On Sep 14, 2024, at 10:59 AM, Shane Hartman ***@***.***> wrote:﻿
Hi, I have used this for a long time.  I pay you for something you run on a server somewhere with paypal (for each home).  Will that still be running and maintained?  I don't think the server code is in this repo?  This plugin is the only thing that kept me with smartthings instead of moving entirely to hubitat or hab.  Thanks!

**Reply by kshartman** [reply]
Cool.  I will keep the subs.  Don't like slower.

**Comment by matheushent** [comment]
I'd be interested in making part of the people who maintain the repo. Maybe an organization would make sense?

**Reply by kshartman** [reply]
I could help.

**Comment by jc-frosty** [comment]
I see that there has not been any updates in a couple of months (9 according to GitHub as of this comments posting).  Is there a definitive fork, especially one 'blessed' by iklein99?

**Reply by bcourbage** [reply]
There's a descendent fork at https://github.com/aziz66/homebridge-smartthings that works well.
